Only show console url / region after configured

kbrock opened the pull request:
- In the login prompt, don't show link to unconfigured console.
- In the summary screen, don't show link to unconfigured appliance.
- If the database is not configured, don't show configuration info
- If the database is not configured, don't show region

https://bugzilla.redhat.com/show_bug.cgi?id=1095558

Extra credit:
- For localhost, was showing blank database hostname, now show localhost.

No database configured

![ac_not_configured_login](https://cloud.githubusercontent.com/assets/1930/6449308/e3c55c4c-c0ed-11e4-8341-97f835bf8926.png)
![ac_not_configured_summary](https://cloud.githubusercontent.com/assets/1930/6449313/e72e9d62-c0ed-11e4-8b3e-74f74becb768.png)

Database configured

![ac_configured_login](https://cloud.githubusercontent.com/assets/1930/6449316/ec876744-c0ed-11e4-9f48-f50675e934bc.png)

![ac_configured_summary](https://cloud.githubusercontent.com/assets/1930/6449318/f10ded56-c0ed-11e4-96c7-5b55e0250a0c.png)

/cc @jrafanie

kbrock commented:
/cc @dajohnso

kbrock commented:
@Fryguy don't know if you're interested in the changes here, but wanted to include you just in case

jrafanie commented on lib/appliance_console.rb:
```diff
@@ -219,11 +220,11 @@ module ApplianceConsole
         Local Database:    #{ApplianceConsole::Utilities.pg_status}
-        EVM Database:      #{dbtype} @ #{dbhost}
-        Database/Region:   #{database} / #{region || 0}
+        EVM Database:      #{ApplianceConsole::DatabaseConfiguration.configured? ? "#{dbtype} @ #{dbhost}" : "not configured"}
```
Should we store  `configured?` in a variable once, each time within this loop.  configured? checks for the database.yml each time it's called, right?

jrafanie commented:
:+1: @kbrock  other than a minor comment

jrafanie commented:
@kbrock thanks for the screenshots... A picture is worth a thousand words :smile:

kbrock commented:
Thanks @jrafanie - cached `configured` - any other suggestions?

jrafanie commented on lib/appliance_console/internal_database_configuration.rb:
```diff
@@ -84,6 +84,10 @@ def post_activation
     end
 
+    def self.installed?
+      File.exist?(DATABASE_DISK_MOUNT_POINT.join("postgresql.conf"))
```
@kbrock is this being used in here?  I don't know that a configuration file existing means PG is installed... What are your thoughts on where we'd use this?

jrafanie commented:
Other than `installed?` comment, looks great!  :sunny:

kbrock commented:
I had used `installed?` to hide/show local server running and removed it.
Removed that code so I've removed `installed?` as well.

@jrafanie Anything else?

jrafanie commented:
LGTM, waiting for travis.  :+1:
